app/views/teams/_team_form.html.erb

```diff
@@ -3,8 +3,10 @@
   <%= f.error_notification %>
   <%= f.input :team_name %>
   <%= f.input :project_level, collection: {Vostok: 'Vostok',
-                                           'Project Gemini'.to_sym => 'Gemini',
-                                           'Apollo 11'.to_sym => 'Apollo 11'}, include_blank: false, required: true %>
+                                           'Project Gemini'.to_sym => 'Project Gemini',
```

Comment:
Change from 'Gemini' to 'Project Gemini' was necessary to ensure the value retrieved from backend matches the values in frontend. Both 'Gemini' and 'Project Gemini' are valid raw values that will be converted into the same enum value in the backend.